<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1714</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="txtLog">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>510</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPic">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>711</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>801</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="labelName1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>630</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>冷却水温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelValue1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>650</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>23.5℃</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>561</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1714</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>系统配置(&amp;S)</string>
    </property>
    <addaction name="actDaqCfg"/>
    <addaction name="separator"/>
    <addaction name="actBcode"/>
    <addaction name="separator"/>
    <addaction name="actComCfg"/>
    <addaction name="separator"/>
    <addaction name="actGlobalCfg"/>
    <addaction name="separator"/>
    <addaction name="actInitDaqCfg"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据采集(&amp;A)</string>
    </property>
    <addaction name="actEnDaq"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>历史记录(&amp;R)</string>
    </property>
    <addaction name="actHisQuery"/>
    <addaction name="separator"/>
    <addaction name="actHisManage"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>数据分析(&amp;D)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDaqCfg"/>
   <addaction name="actEnDaq"/>
   <addaction name="actHisQuery"/>
   <addaction name="actAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actDaqCfg">
   <property name="icon">
    <iconset>
     <normaloff>:/new/images/images/setting.png</normaloff>:/new/images/images/setting.png</iconset>
   </property>
   <property name="text">
    <string>测点配置</string>
   </property>
   <property name="toolTip">
    <string>配置测点信息...</string>
   </property>
  </action>
  <action name="actEnDaq">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/images/images/recording.ico</normaloff>:/new/images/images/recording.ico</iconset>
   </property>
   <property name="text">
    <string>启动采集</string>
   </property>
   <property name="toolTip">
    <string>开始采集数据</string>
   </property>
  </action>
  <action name="actHisQuery">
   <property name="icon">
    <iconset>
     <normaloff>:/new/images/images/search.png</normaloff>:/new/images/images/search.png</iconset>
   </property>
   <property name="text">
    <string>数据查询</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/new/images/images/help.png</normaloff>:/new/images/images/help.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actBcode">
   <property name="text">
    <string>B码对时</string>
   </property>
   <property name="toolTip">
    <string>B码对时配置</string>
   </property>
  </action>
  <action name="actComCfg">
   <property name="text">
    <string>串口配置</string>
   </property>
  </action>
  <action name="actGlobalCfg">
   <property name="text">
    <string>通用配置</string>
   </property>
  </action>
  <action name="actHisManage">
   <property name="text">
    <string>数据管理</string>
   </property>
  </action>
  <action name="actInitDaqCfg">
   <property name="text">
    <string>初始化测点信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>856</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
